.github/workflows/main.yaml

```diff
@@ -27,9 +27,12 @@ jobs:
         scripts/alpine-setup.sh
         make test
   build:
-    name: Build
+    name: build
     runs-on: ubuntu-latest
-    container: golang:1.16.2-alpine3.12
+    container: golang:1.16.1-alpine3.12
```

Comment:
I merged this yesterday, can you please rebase so that we're not reverting this change.